securedrop/tests/functional/pageslayout/test_journalist.py

```diff
@@ -16,92 +16,125 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 import pytest
-import tests.functional.pageslayout.functional_test as pft
-from tests.functional import journalist_navigation_steps, source_navigation_steps
+from tests.functional.app_navigators import JournalistAppNavigator
+from tests.functional.pageslayout.functional_test import list_locales
+from tests.functional.pageslayout.screenshot_utils import save_screenshot_and_html
 
 
+@pytest.mark.parametrize("locale", list_locales())
```

Comment:
Below I merged some of the "screenshot-taking" tests, so that the test suite would run faster.